x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/transforms_stats_bar.tsx

```diff
@@ -87,12 +100,50 @@ function createTranformStats(transformsList: TransformListRow[]) {
   return transformStats;
 }
 
-interface Props {
+interface TransformStatsBarProps {
+  transformNodes: number;
   transformsList: TransformListRow[];
 }
 
-export const TransformStatsBar: FC<Props> = ({ transformsList }) => {
-  const transformStats: TransformStatsBarStats = createTranformStats(transformsList);
+export const TransformStatsBar: FC<TransformStatsBarProps> = ({
+  transformNodes,
+  transformsList,
+}) => {
+  const { esNodeRoles } = useDocumentationLinks();
+
+  const transformStats: TransformStatsBarStats = createTranformStats(
+    transformNodes,
+    transformsList
+  );
 
-  return <StatsBar stats={transformStats} dataTestSub={'transformStatsBar'} />;
+  return (
+    <>
+      <StatsBar stats={transformStats} dataTestSub={'transformStatsBar'} />
+      {transformNodes === 0 && (
+        <>
+          <EuiSpacer size="m" />
+          <EuiCallOut
+            title={
+              <FormattedMessage
+                id="xpack.transform.transformNodes.noTransformNodesMessage"
+                defaultMessage="There are no transform nodes available. {learnMoreLink}"
```

Comment:
Comparing to the ML warning, might be worth adding `You will not be able to create or run transforms.` to this message?

Reply:
Fixed in c47595c.